src/components/Reports/ReportsTrafficSources/ReportsTrafficSources.js

```diff
@@ -45,28 +46,31 @@ const ReportsTrafficSources = function({
         <h4 className="title-ranking">
           <FormattedMessage id="trafficSources.title" />
         </h4>
-        <div>
+        <Container>
           {state.loading ? (
             <Loading />
           ) : state.trafficSources ? (
             state.trafficSources.items.map((trafficSource, index) => (
-              <div key={index}>
-                <span>{trafficSource.sourceName}</span>
-                <span>
-                  {trafficSource.quantity} (
-                  <FormattedNumber
-                    value={trafficSource.quantity / state.trafficSources.total}
-                    {...numberFormatOptions}
-                  />
-                  )
-                </span>
-              </div>
+              <TrafficSourceContainer key={index}>
```

Comment:
The only thing I find odd is that this components only functionality is styling. When UI changes are ready you will need to remove this components right? What about tests? Will you be needing to import this components in too?

Reply:
I've talked through this with @fcoronelmakingsense apart. Some styles will remain in webapp and some not.